docs/apache-spark-with-minio.md

```diff
@@ -1,10 +1,10 @@
-# Apache Spark with Minio Server [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+# Apache Spark with MinIO Server [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-Apache Spark is a fast and general engine for large-scale data processing. In this recipe we'll see how to launch jobs on Apache Spark-Shell that reads/writes data to a Minio server.
+Apache Spark is a fast and general engine for large-scale data processing. In this recipe we'll see how to launch jobs on Apache Spark-Shell that reads/writes data to a MinIO server.
 
 ## 1. Prerequisites
 
-- Install Minio Server from [here](http://docs.minio.io/docs/minio-quickstart-guide).
+- Install MinIO Server from [here](http://docs.minio.io/docs/minio-quickstart-guide).
```

Comment:
Two things here. 
1. We are not consistent with where we want the user to download `minio server` from. Previously we redirect them to go to `www.minio.io` and here we are directing them to the doc site. 

2. The docs site can be `http://docs.min.io`

Reply:
Please approve this PR and then I will change the domains next.